app/assets/javascripts/discourse/helpers/application.js.es6

```diff
@@ -41,6 +41,13 @@ registerUnbound("number", (orig, params) => {
   if (n.toString() !== title.toString() && addTitle) {
     result += " title='" + Handlebars.Utils.escapeExpression(title) + "'";
   }
+  if (params.ariaLabel) {
+    result +=
+      " aria-label='" +
+      Handlebars.Utils.escapeExpression(params.ariaLabel) +
+      "'";
```

Comment:
I don't really like this string spanning 4 lines... Maybe we could write it this way?

```javascript
const ariaLabel = Handlebars.Utils.escapeExpression(params.ariaLabel);
result += ` aria-label='${ariaLabel}'`;
```